src/main/java/org/myrobotlab/service/Agent.java

```diff
@@ -98,16 +98,18 @@
   boolean autoCheckForUpdate = false;
 
   Set<String> possibleVersions = new TreeSet<String>();
+  
+  final static String REMOTE_BUILDS_URL_HOME = "http://build.myrobotlab.org:8080/job/myrobotlab-multibranch/";

```

Comment:
we should make this a default value, and allow users to specify a system property to override this url...  (maybe?)